src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+
+
+    public static String getRandomColor() {
```

Comment:
Redundant new lines. Try to maintain consistency across all classes.
Also, static methods are generally a bad practice. Avoid that unless stated otherwise in the task requirements.

![image](https://github.com/kacper-kukula/my-test-task/assets/105852806/e94247c2-c909-4f4a-a2f0-99b5d533424a)
![image](https://github.com/kacper-kukula/my-test-task/assets/105852806/7e961b7f-0c54-44f8-b3bb-f656ef17c012)